### PR TITLE
add documentation to home page and add drivers to header

### PR DESCRIPTION
**Pull request by jchris**
closes https://github.com/fauna/sales-engineering/issues/562

copy is:

## Welcome to FaunaDB

The developer dashboard allows you to browse and create databases, inspect and define schemas and instances, and explore your data with the interactive query console.

### Dashboard Quickstart

Follow these steps to get comfortable using this dashboard. After each step click the Fauna logo in the top-left corner to get back to this list.

1. Databases are containers for data. You can create as many databases as you like, and they can be nested. Click Create Database in the sidebar to create one.
2. A class is required before you can save data. Each database may contain multiple classes. Instances of a class usually have common structure, but it's OK for their structure to vary. To create a class, navigate via the sidebar to the database you just created, then click Create Class. You probably want to check the "Create Class Index" box on the form.
3. Now you are ready to save some data. To create an instance of your class, browse to your database, and then select your class in the sidebar. Modify the text area so it has more interesting data than an empty {} and click Create Instance.
4. When you created the class, if you checked "Create Class Index" there will also be an index called all_MYCLASS. Click it to browse all the instances in the class. Clicking an instance in the result set will reveal the history of that instance.
5. Last step: pop open the query console by clicking "Toggle Query Console" and run the query there which will list your index. Play around with the query console with other queries. Here's one to add a field to your instance (you'll want to use the Ref listed on the instance you just browsed in the main UI.):

```
q.Update(q.Ref("classes/myclass/159441342217322496"), {data: {foo : "bar"}})
```

Congratulations, you seem to be getting the hang of this. Check out the links tutorials and documentation in the header. If you need help, click the chat bubble in the lower right, we love feedback.

### CODE REVIEW

File: src/new-dash/get-started/index.js [hunk 1]
```diff
@@ -0,0 +1,41 @@
+import React from 'react';
```
Comment (erickpintor) [inline src/new-dash/get-started/index.js]: We're not using semicolon.

Comment (jchris) [inline src/new-dash/get-started/index.js]: yes I like the idea of getting it somewhere it displays before the JavaScript parses.

Comment (erickpintor) [inline src/new-dash/get-started/index.js]: I'm sorry. I didn't understand your comment. Are you talking about the semicolon?

Comment (jchris) [inline src/new-dash/get-started/index.js]: no I accidentally put it in the wrong place. I'm updating the PR to make the content appear before discoverKeyType runs. It'll still be a module, I'm not sure exactly the best way to make it an HTML file. But it'll render fast.

Comment (erickpintor) [inline src/new-dash/get-started/index.js]: Maybe it's not worth investing the time to add it at the old dashboard. I'm expecting the new one to be released this week.

Making it a module seems odd to me. I would place it in a file at `src/new-dash` for now. If we get more pages like this, we should think about a place for them.

**Comment by jchris** [comment]
This wil render a little sooner now. But on the new-dash there's a modal dialog during loading anyway.

**Comment by erickpintor** [comment]
Since this code is in production already and I'm getting ready to deploy the new dashboard, I'm merging this.

I've removed the code that attempts to render the page before the fauna client is available. There is a popup that prevents the user to see the content anyway. Also, rendering client dependent components without a client could cause some harm. We just assume the client is available so we don't have to `if (!this.props.client)` everywhere.

**Pull request merged**